### DISCUSSION TITLE
Carousel - Disable draggable

### DISCUSSION THREAD

**Discussion by bckthomas** [fancyapps/ui#74] [Q&A]
Hello
Can you tell me how I can disable the is-draggable class for carousel ?
Thanks

**Comment by fancyapps** [comment]
Hi,

Carousel is built around Panzoom, and this is how you set [Panzoom options](https://fancyapps.com/docs/ui/panzoom/options) including `draggableClass` that is used to set `is-draggable` class:

```JS
const myCarousel = new Carousel(document.querySelector("#myCarousel"), {
  slides: [
    { html: "1" },
    { html: "2" },
    { html: "3" },
    { html: "4" },
    { html: "5" },
  ],
  Panzoom: {
    draggableClass: false,
    draggingClass: false,
  },
});
```

Demo - https://fancyapps.com/playground/Ce
